contents/tutorials/nextjs-error-monitoring.md

```diff
@@ -294,4 +294,52 @@ Once you’ve set up error capture in your app, you can head to the [error track
   classes="rounded"
 />
 
-You can click into any of these errors to get more details on them, including a stack trace as well as archive, resolve, or suppress them. On top of this, you can analyze `$exception` events like you would any event in PostHog, including setting up [trends](/docs/product-analytics/trends/overview) for them and querying them with [SQL](/docs/product-analytics/sql).
\ No newline at end of file
+You can click into any of these errors to get more details on them, including a stack trace as well as archive, resolve, or suppress them. On top of this, you can analyze `$exception` events like you would any event in PostHog, including setting up [trends](/docs/product-analytics/trends/overview) for them and querying them with [SQL](/docs/product-analytics/sql).
+
+## 5. Uploading source maps
+
+import CLIDownload from "../docs/error-tracking/_snippets/cli/download.mdx"
+import CLIAuthenticate from "../docs/error-tracking/_snippets/cli/authenticate.mdx"
+import CLIInject from "../docs/error-tracking/_snippets/cli/inject.mdx"
+import CLIUpload from "../docs/error-tracking/_snippets/cli/upload.mdx"
+
+PostHog uses source maps to show unminified code in your stack traces (so you can find where the errors are coming from). Next.js disables source maps by default during production builds to prevent you from leaking your source on the client, but you can opt-in with the `productionBrowserSourceMaps` configuration flag.
+
+To enable this, we set a environment variable to control this so we only generate source maps when building locally (or if you are running a build in CI).
+
+```file=.env.local
+NEXT_PUBLIC_POSTHOG_SOURCE_MAPS=true
+```
+
+Once we have the environment variable set, we can configure Next.js to generate source maps.
+
+```js file=next.config.js 
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  productionBrowserSourceMaps: process.env.GENERATE_SOURCEMAPS === 'true'
```

Comment:
ditto above comment, not sure if this should match `NEXT_PUBLIC_POSTHOG_SOURCE_MAPS`